### ISSUE TITLE
Enable oidc-authservice repository CI for power(ppc64le) architecture.

### ISSUE THREAD

**Issue by amitmukati-2604** [arrikto/oidc-authservice#104]
**Is this a bug report or feature request?**
* Feature Request
Enable the OIDC-authservice repository CI for power(ppc64le) architecture. Any inputs on how we can enable CI for power(ppc64le)? As the current Yaml file in the repository does not explain much about architecture-specific configuration, hence not getting a clear idea. Can you help me with the oidc-authservice CI enabling for power(ppc64le) architecture?

**What should the feature do:**
 This will enable CI of OIDC-authservice for power(ppc64le) which builds and push images for both amd64 and ppc64lee.

**Additional Information:**

I have created one workflow which will just build and push the images for amd64 and ppc64le. Below is the link to the workflow and pushed images.
 Link to the workflow:- https://github.com/amitmukati-2604/oidc-authservice/actions/runs/3674045572/jobs/6211797806
 Link to pushed images:- https://hub.docker.com/r/amitmukati2604/oidc-authservice/tags

**Comment by lehrig** [comment]
@kimwnasptd, can you provide a hint for how to get started? How/where is CI handled for this project and to whom should we talk to?

**Comment by pranavpandit1** [comment]
@athamark : are these tags intended for this issue (does not seem to be relevant with issue reference)?

**Comment by athamark** [comment]
@pranavpandit1 you are right. These commits were meant to reference GitHub-PR: #105 instead of this issue. I force-pushed a branch with the correct references. Sorry for the confusion.

**Comment by pranavpandit1** [comment]
@kimwnasptd: I am also interested, any suggestions on where to start with?
